Triphouse-project/style.css

```diff
@@ -0,0 +1,46 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+.container {
+    margin: 0px auto;
+  text-align: center;
+  max-width: 1200px;
+}
+.offer_title {
+    margin-bottom: 200px;
+    position: relative;
+}
+
+.offer_title::after {
+ content: '';
+ position: absolute;
+ display: block;
+ width: 100px;
+height: 10px;
+ margin: 0px auto;
+ top: 100px;
+ left: 0;
+ right: 0;
+ background: #f5bd41;
+}
+
+.offer_items {
+    display: flex;
+    justify-content: space-between;
+}
+
+@media (max-width: 500px) {
+    .container .offer_items {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  .offer__item-img {
```

Comment:
Add some height and margin bottom, will be prettier.
![2021-04-08_22h17_14](https://user-images.githubusercontent.com/19935417/114083857-2fe6ee80-98b8-11eb-948f-9795718e3c9b.png)